docs/src/reference/compilation.rst

```diff
@@ -492,6 +492,59 @@ on end user side as it hooks into their import system.  The best way
 to cater for end users is to provide pre-built binary packages in the
 `wheel <https://wheel.readthedocs.io/>`_ packaging format.
 
+
+Arguments
+---------
+
+The function ``pyximport.install()`` can take several arguments to
+influence the compilation of Cython or Python files.
+
+.. autofunction:: pyximport.install
+
+
+Dependency Handling
+--------------------
+
+Since :mod:`pyximport` does not use `cythonize()` internally, it currently
```

Comment:
Right, and that is totally a bug. It should _really_ use cythonize(). Definitely something to list under "limitations" below.

Reply:
I've put a paragraph in "limitations" about it.